src/unix/notbsd/android/mod.rs

```diff
@@ -562,6 +565,7 @@ pub const TIOCMBIC: ::c_int = 0x5417;
 pub const TIOCMSET: ::c_int = 0x5418;
 pub const FIONREAD: ::c_int = 0x541B;
 pub const TIOCCONS: ::c_int = 0x541D;
+pub const CLONE_NEWCGROUP: ::c_int = 0x02000000;
```

Comment:
I agree, it definitely should be merged.

The constant is defined on all architectures: https://github.com/torvalds/linux/blob/master/include/uapi/linux/sched.h